packages/siop-oid4vp/lib/rp/RP.ts

```diff
@@ -196,13 +198,13 @@ export class RP {
         correlationId,
       })
       correlationId = verifyAuthenticationResponseOpts.correlationId ?? correlationId
-      void this.emitEvent(AuthorizationEvents.ON_AUTH_RESPONSE_RECEIVED_SUCCESS, {
+      await this.emitEvent(AuthorizationEvents.ON_AUTH_RESPONSE_RECEIVED_SUCCESS, {
```

Comment:
First why is this needed. This makes the events sync. Second. If you change it here, then at least change it everywhere in this file